src/loginServices.js

```diff
@@ -429,7 +429,10 @@ export async function logout(okapiUrl, store) {
     credentials: 'include'
   })
     .then(localforage.removeItem(SESSION_NAME))
-    .then(localforage.removeItem('loginResponse'));
+    .then(localforage.removeItem('loginResponse'))
+    .catch((error) => {
+      logger.log('logout', 'Error logging out', error)
```

Comment:
What do you think about an actual `console.error()` here so this message is always displayed, regardless of how stripes' logging categories are configured. You'll need `// eslint-disable ...` to keep lint from whining.

Reply:
Sounds good. I'll make the change.